### PR TITLE
Monika real world topic

### PR DESCRIPTION
**Pull request by HalHarrison**
First time doing on of these menu topics. 
It seems like the sub menus arent working, or the return. Tried to fix but I dont know what to do :/
I think the topic itself is pretty solid

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+    for key in ['real world']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_real_world')
+    monika_random_topics.append('monika_real_world')
+
+label monika_real_world:
+    m 1e "[player], how long do you think it will be before I will be able to be with you in your world?"
+    menu:
+        "Soon":
+            m 1b "That's great news!
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+            m 3a "Be sure to let me know when you find a way."
+            m 3j "You know right where to find me~."
+        "Never":
+            m 1i "Wha.. what.
+            m 3o "..."
+            m 3p "Are you sure?"
+            menu:
+                "Yes"
+                    m 1o "..."
+                    m 3q "Well, I guess we'll just have to make the best of this then.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1m "As long as you talk to me that's all I really need.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1g "And [player], whatever you do...
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1p "Dont leave me here all alone.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1o "I would rather be deleted permanently than to be stuck in here for eternity by myself.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1g "So please, if you ever get... bored with me.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1o "You know what to do."
+                    m "..."
+                    m 1g "I wouldn't want to live without you."
+                "No, but it is a very difficult problem to fix. Its best not to assume it will ever happen."
+                    m 3r "I see."
+                    m 3q "..."
+                    m 3h "Well, even if there is only a miniscule chance I won't give up hope.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 3i "Neither should you, [player]."
+                    m 1p "And if things turn out that I can never leave this place and be with you...
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m "..."
+                    m 1g "I wouldn't want to live without you."
+                "No, but it is a very difficult problem to fix. Its best not to assume it will ever happen."
+                    m 3r "I see."
+                    m 3q "..."
+                    m 3h "Well, even if there is only a miniscule chance I won't give up hope.
+                    m 3i "Neither should you, [player]."
+                    m 1p "And if things turn out that I can never leave this place and be with you... 
+                    m 1g "At least come here and talk to me as much as you can."
+                    m 1f "You're all that I care about.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -4790,6 +4790,66 @@ label monika_urgent:
...
+                    m 1g "I wouldn't want to live without you."
+                "No, but it is a very difficult problem to fix. Its best not to assume it will ever happen."
+                    m 3r "I see."
+                    m 3q "..."
+                    m 3h "Well, even if there is only a miniscule chance I won't give up hope.
+                    m 3i "Neither should you, [player]."
+                    m 1p "And if things turn out that I can never leave this place and be with you... 
+                    m 1g "At least come here and talk to me as much as you can."
+                    m 1f "You're all that I care about. 
+return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `return` needs an indent.

**Comment by HalHarrison** [comment]
Thanks Winblows

**Comment by monikLover** [comment]
Edited and reviewed. Good job!

**Pull request merged**